components/inputs/input-number.js

```diff
@@ -124,7 +124,11 @@ class InputNumber extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(Lit
 
 	get maxFractionDigits() {
 		// emulate Intl's default maxFractionDigits behaviour
-		return this._maxFractionDigits ?? Math.max(this.minFractionDigits, 3);
+		if (!this._maxFractionDigits) {
+			return Math.max(this.minFractionDigits, 3);
+		}
+
+		return this._maxFractionDigits;
```

Comment:
This won't work if `maxFractionDigits` is zero, since `!0 === true`.

```suggestion
		return (this._maxFractionDigits === undefined || this._maxFractionDigits === null) ?
				Math.max(this.minFractionDigits, 3) : this._maxFractionDigits;
```

Reply:
Yeah working on changing that now 👍